expressEngine/lib/service/db.js

```diff
@@ -1,27 +1,31 @@
-class DbOperation {
-  count = (collection, data = {}) => {
-    return collection.countDocuments(data);
-  }
+import BaseService from './base';
+import { error } from '../../cms/user';
 
-  create = (collection, data = {}) => {
-    return collection.create(data);
-  }
+const baseService = new BaseService();
+class DBOperation {
+  count = (collection, data = {}) => collection.countDocuments(data)
 
-  find = (collection, data = {}, projection = {}) => {
-    return collection.find(data, projection);
-  }
+  create = (collection, data = {}) => collection.create(data)
 
-  findOne = (collection, data = {}, projection = {}) => {
-    return collection.findOne(data, projection);
-  }
+  find = (collection, data = {}, projection = {}) => collection.find(data, projection)
 
-  deleteMany = (collection, data) => {
-    return collection.deleteMany(data);
-  }
+  findOne = (collection, data = {}, projection = {}) => collection.findOne(data, projection)
 
-  deleteOne = (collection, data) => {
-    return collection.deleteOne(data);
+  findAll = ({
```

Comment:
@yogeshsinghh What is the use of findAll, when we already have find.

Reply:
sir, I created a new method findAll for just because if we use find then there might be a lot of conflicts for another PR that uses find methods because findAll methods which take the arguments as an object